Content.Shared/Audio/Jukebox/SharedJukeboxSystem.cs

```diff
@@ -1,8 +1,24 @@
 using Robust.Shared.Audio.Systems;
+using Content.Shared.Emag.Systems; //imp
 
 namespace Content.Shared.Audio.Jukebox;
 
 public abstract class SharedJukeboxSystem : EntitySystem
 {
     [Dependency] protected readonly SharedAudioSystem Audio = default!;
+
+    //imp for everything below this line ------------------
```

Comment:
somebodys gonna be so mad at this in 5 years when upstream changes some jukebox functionality